build.gradle

```diff
@@ -1,5 +1,5 @@
 buildscript {
-	repositories { maven { url 'https://plugins.gradle.org/m2/' } }
+	repositories { maven { url 'https://plugins.gradle.org/m2/' }}
```

Comment:
Sorry, but the formatter always replaces nested closures by }  } and then starting the ping-pong, unless I reformat it to }}.
Furthermore it cannot convinced to leave closures with one entry split over multiple lines.